examples/panel-basic/cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+      "target": "es5",
+      "lib": ["es5", "dom"],
+      "types": ["cypress"]
+    },
+    "include": ["**/*.ts"]
+  }
```

Comment:
**Question:** could this use (depend on) the `tsconfig.json` at the root of the plugin directory?

Reply:
It certainly can. Have updated it.